src/ts/components/subtitleoverlay.ts

```diff
@@ -49,6 +43,11 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     this.subtitleManager = subtitleManager;
 
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
+      // Sanitize invalid positioning data
+      // Sometimes the positions are undefined, we assume them to be zero
+      event.position.row = event.position.row || 0;
+      event.position.column = event.position.column || 0;
```

Comment:
Thanks, that happens when writing a "quickfix"... Unfortunately I have to do that here in this handler, otherwise the cues get wrong IDs. That can be fixed in the cea-608 specific handler, but only in cueEnter, cueExit does not have any idea about the wrong and corrected data, so I cannot retrieve the cues by ID there.